packages/kit/src/runtime/client/remote-functions/form.svelte.js

```diff
@@ -239,6 +239,11 @@ export function form(id) {
 
 				const form_data = new FormData(form);
 
+				const submitter_name = event.submitter?.getAttribute('name');
+				if (submitter_name) {
+					form_data.append(submitter_name, event.submitter?.getAttribute('value') ?? '');
+				}
```

Comment:
TIL there's an optional `submitter` parameter we can pass to `FormData` instead of implementing this ourselves. https://developer.mozilla.org/en-US/docs/Web/API/FormData/FormData#submitter